extensions/reports/timeout_report.go

```diff
@@ -0,0 +1,13 @@
+package reports
+
+import (
+	management "github.com/rancher/shepherd/clients/rancher/generated/management/v3"
+	"github.com/rancher/shepherd/pkg/wait"
+	"github.com/sirupsen/logrus"
+)
+
+func TimeoutReport(cluster *management.Cluster, err error) {
```

Comment:
Can you add go doc comment here?

Reply:
Agreed,  also I have a question: will this work for v1 clusters?

Reply:
Good point Ugur. You might need to have two functions. One for rke1 and one rke2/k3s

Reply:
Yes, I can

Reply:
I'll crate these new functions